pset-functions.js

```diff
@@ -208,6 +319,24 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const wordy = 'rune'
+
+function doubleLetters(word) {
```

Comment:
@kcycle great approach, Using a variable to represent the "last letter" make your solution easy to navigate. Also, great use of a `return` statement to exit out of the function as soon as a match is found